frankenstein/chunk_store.go

```diff
@@ -121,7 +121,7 @@ func NewAWSChunkStore(cfg ChunkStoreConfig) (*AWSChunkStore, error) {
 	return &AWSChunkStore{
 		dynamodb:   dynamodb.New(session.New(dynamoDBConfig)),
 		s3:         s3.New(session.New(s3Config)),
-		memcache:   cfg.MemcacheClient,
```

Comment:
we don't see to populate cfg in here?  could do that and not have our own reference to chunkCache, or drop cfg from the struct.